packages/auth/seeds/development/01_grants.js

```diff
@@ -11,8 +11,9 @@ exports.seed = function (knex) {
           startMethod: ['redirect'],
           finishMethod: 'redirect',
           finishUri: 'https://example.com/finish',
+          client: 'https://backend/accounts/gfranklin',
+          clientKeyId: 'keyid-742ab7cd-1624-4d2e-af6e-e15a71638669',
```

Comment:
Ideally, this would be the local/postman client, especially if we are keeping the P2P postman request collection, but that is still a ongoing discussion.

At first, I made this grant's client be equal to the fynbos rafiki instance, to sort of give a demonstration of having a "third/different" client facilitate the p2p between gfranklin and pfry, but decided to leave these grants "bound" to the respective users' payment pointers.